ozaria/site/views/play/level/LevelPlaybackView.coffee

```diff
@@ -303,6 +303,9 @@ module.exports = class LevelPlaybackView extends CocoView
   onTogglePlay: (e) ->
     e?.preventDefault?()
     return if @shouldIgnore()
+    playing = store.state.game.playing
+    if not playing
+      Backbone.Mediator.publish('tome:manual-cast', {realTime: true})
```

Comment:
Does this piece of code need to execute for all levels, or just capstone?
Also, I think the UI of the play button/the whole LevelPlayBackView is different for the capstone, is that going to be handled in a separate PR?

Reply:
For other levels, it would usually go through the "Run" button. As this area is swapped out with the UX for the scrubber area, this can move there. It will be handled in a separate PR.

Reply:
The state of the buttons and the cast'ing is tricky between 4 states: 
- regular levels in coco
- game dev levels in coco
- regular levels in ozaria
- game dev (or capstone) levels in ozaria

This is handled in the next PR for the play button.